blacksauce/src/styles/_nav.scss

```diff
@@ -0,0 +1,134 @@
+// Variables
+$display-nav: 520px;
+
+// JS action classes
+[data-visible="false"] {
+    display: none;
+}
+
+.site-header {
+    display: flow-root;
+}
+
+.site-nav {
+    $top-height: 68px;
+
+    &__trigger {
+        position: absolute;
+        top: 1em;
+        right: 1em;
+
+        &[aria-expanded="true"] {
+            .menu-icon__line {
+                $yValue: 10px;
+                $deg: 45deg;
+
+                &:nth-child(1) {
+                    transform: translateY($yValue) rotate($deg);
+                }
+
+                &:nth-child(2) {
+                    opacity: 0;
+                }
+
+                &:nth-child(3) {
+                    transform: translateY(-$yValue) rotate(-$deg);
+                }
+            }
+        }
+
+        @media screen and (min-width: $display-nav) {
+            display: none;
+        }
+    }
+
+    &__list {
+
+        @media screen and (max-width: ($display-nav - 1px)) {
+            // background-color: rgba(rgb(0, 0, 0), 0.75);
+            background-color: black; // @feedback I am trying to make the background of the links solid black or at least not do the funky thing in which my text is takes presedent visually over the links (the background seems to only cover the image not the text). Also there is the corner of that one button that is so much brighter then the rest and I am fairly confused by that.
+            flex-direction: column;
+            height: 0;
+            position: absolute;
+            top: $top-height;
+            right: 0;
+            left: 0;
+            width: 100vw;
+
+            &[data-visible="true"] {
+                animation: expand 0.3s ease-in-out forwards; // see the expand animation below
+                // this changes the height and opacity of the mobile menu
+                display: flex;
+            }
```

Comment:
Add `z-index: 90;` after the display property here and that should solve the funkiness with your main content text appearing on top of the open mobile nav. 👍

Reply:
This does not seem to fix my problem unless I am not using it correctly.

Reply:
Just kidding, I was not watching my scss, sorry about that.